app/controllers/IncomeSourcesController.scala

```diff
@@ -60,4 +62,19 @@ class GetIncomeSourcesController @Inject()(getIncomeSourcesService: GetIncomeSou
       case Left(error) => Status(error.status)(error.toJson)
     }
   }
+
+  def refreshIncomeSource(nino: String, taxYear: Int): Action[AnyContent] = authorisedAction.async { implicit user =>
```

Comment:
Nino is never used. Is this a mistake? Should it be removed from the rest end point path as we use the user?
Same issue with getIncomeSourcesFromSession(...)

Reply:
it's used in the auth action from the url, but not used explicitly